jjb/anax/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent any
```

Comment:
agent needs to be `ubuntu2204-builder-aws-1c-2g`. See my comment on the `node` setting in the jjb file below.